Sources/HaCWebsiteLib/ViewModels/LandingPage.swift

```diff
@@ -18,7 +18,37 @@ struct LandingPage {
               El.Div[Attr.className => "LandingIntroduction__tagLine"].containing("Cambridge's student tech society")
             ),
             El.P.containing("We build things, and help other people build things. We also have a bunch of fun"),
-            El.A[Attr.href => "#", Attr.className => "LandingIntroduction__link"].containing("Find out more")
+            El.A[Attr.href => "#", Attr.className => "LandingIntroduction__link"].containing("Find out more"),
+            El.Div[Attr.className => "LandingIntroduction__titleIconRow"].containing(
+              El.A[Attr.href => "https://github.com/hackersatcambridge/", Attr.className => "LandingIntroduction__titleLinkIcon"].containing(
+                El.Img[
+                  Attr.className => "LandingIntroduction__titleLinkIconImg",
+                  Attr.src => "/static/images/github_icon.png",
```

Comment:
Is it possible to get these as SVGs? They reduce load and mean we don't have to worry so much about retina displays et al